languages/woocommerce-postnl-nl_NL.po

```diff
@@ -1108,26 +1108,26 @@ msgstr ""
 #~ "in ontvangst is genomen door de geadresseerde. Een veilig gevoel."
 
 #~ msgid ""
-#~ "When you connect the customer email, MyParcel can send a Track&Trace "
+#~ "When you connect the customer email, PostNL can send a Track&Trace "
 #~ "email to this address. In your <a href=\"https://backoffice.myparcel.nl/"
-#~ "ttsettingstable\">MyParcel backend</a> you can enable or disable this "
+#~ "ttsettingstable\">PostNL backend</a> you can enable or disable this "
 #~ "email and format it in your own style."
 #~ msgstr ""
-#~ "Wanneer u het emailadres van de klant koppelt, kan MyParcel een "
+#~ "Wanneer u het emailadres van de klant koppelt, kan PostNL een "
 #~ "Track&Trace email versturen naar dit adres. In uw <a href=\"https://"
-#~ "backoffice.myparcel.nl/ttsettingstable\">MyParcel backend</a> kunt u deze "
+#~ "backoffice.myparcel.nl/ttsettingstable\">PostNL backend</a> kunt u deze "
 #~ "email in- of uitschakelen en opmaken in uw eigen stijl."
 
 #~ msgid "Productnaam"
 #~ msgstr "Productnaam"
 
 #~ msgid ""
```

Comment:
Vanaf regel 1124 t/m 1128:
Dit is niet mogelijk voor een PostNL klant. Dit kan weg. Maar eerst zoeken waar dit wordt gebruikt. Ook engelse tekst opzoeken en weghalen

Reply:
Cart is aangemaakt om heel "Koppel emailadres klant" weg te halen, dus dit zal ook meegenomen worden.